packages/stark-ui/assets/styles/_variables.scss

```diff
@@ -1,12 +1,49 @@
-$border-radius: 2px;
+$color-theme: (
+  accent: (
+    color: #8f0039,
+    contrast: $light-primary-text
+  ),
+  alert: (
+    color: #d32f2f,
+    contrast: $light-primary-text
+  ),
+  alt: (
+    color: #323232,
+    contrast: $light-primary-text
+  ),
+  neutral: (
+    color: #bfbfbf,
+    contrast: $light-primary-text
+  ),
+  primary: (
+    color: #0076c8,
+    contrast: $light-primary-text
+  ),
+  success: (
+    color: #4caf50,
+    contrast: $light-primary-text
+  ),
+  warning: (
+    color: #ff9800,
+    contrast: $light-primary-text
+  ),
+  white-transp: (
+    color: rgba(255, 255, 255, 0.87),
+    contrast: $dark-primary-text
+  )
+);
+
+@if variable-exists(stark-color-theme) {
+  $color-theme: map-merge($color-theme, $stark-color-theme);
+}
 
 $primary-palette: (
   50: #e3f2fd,
```

Comment:
Shouldn't all the shades be also customizable? Otherwise it might be that the color will be completely different when using shades 500 and any other 😕

Reply:
yes they will have the choice, either to use the simple array, either to override the full palettes. It can be heavy to ask them to override all the palettes, so I think we should keep a simple version of the map. Of course we can add new color as we go on.